src/index.js

```diff
@@ -1,8 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
 
-import HomeComponent from './app/Home/components';
+import Router from './routes';
+import configureStore from './store';
 
 import './styles/main.scss';
 
-ReactDOM.render(<HomeComponent />, document.getElementById('root'));
+const store = configureStore();
```

Comment:
@alexmochu `configureStore()` takes initial state. You have not provided that here

Reply:
@Chris-Maina I have configured the Store here --> https://github.com/andela/andela-societies-frontend/pull/170/files#diff-14

This is where I have provided the initial state.